corehq/apps/linked_domain/local_accessors.py

```diff
@@ -10,6 +10,30 @@
 from corehq.apps.users.models import SQLUserRole
 from corehq.apps.users.views.mobile import UserFieldsView
 from corehq.apps.integration.models import DialerSettings, GaenOtpServerSettings, HmacCalloutSettings
+from corehq.apps.reports.models import TableauServer, TableauVisualization
+
+
+def get_tableau_visualizaton(domain):
+    server, created = TableauServer.objects.get_or_create(domain=domain)
+    visualizaton, created = TableauVisualization.objects.get_or_create(domain=domain, server=server)
```

Comment:
I'm curious how this will work when there is more than one visualization in the domain associated with the same server